zypp/KeyRing.h

```diff
@@ -26,6 +26,9 @@
 #include <zypp/PublicKey.h>
 #include <zypp/KeyRingContexts.h>
 
+#include <zypp-core/zyppng/async/AsyncOp>
+#include <zypp-core/zyppng/pipelines/Expected>
+
```

Comment:
/zyppng/ includes should not be exposed here (and are AFAICS not needed).

Reply:
Done